<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotifierWindow</class>
 <widget class="QWidget" name="NotifierWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>176</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="2">
    <widget class="QWidget" name="show_panel" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="button_show">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../common/resources/common.qrc">
          <normaloff>:/img/back.svg</normaloff>:/img/back.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="title" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="button_hide">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../common/resources/common.qrc">
          <normaloff>:/img/forward.svg</normaloff>:/img/forward.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_title">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">Title</string>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::RichText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="content" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="tree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="indentation">
         <number>0</number>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>true</bool>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="headerCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">0</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="toolbar" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>194</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="button_lock_mouse">
           <property name="toolTip">
            <string>Lock mouse</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="../../common/resources/common.qrc">
             <normaloff>:/img/computer-mouse.svg</normaloff>:/img/computer-mouse.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="button_lock_keyboard">
           <property name="toolTip">
            <string>Lock keyboard</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="../../common/resources/common.qrc">
             <normaloff>:/img/keyboard.svg</normaloff>:/img/keyboard.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="button_pause">
           <property name="toolTip">
            <string>Pause</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="../../common/resources/common.qrc">
             <normaloff>:/img/pause.svg</normaloff>:/img/pause.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="button_stop">
           <property name="toolTip">
            <string>Disconnect all</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="../../common/resources/common.qrc">
             <normaloff>:/img/stop.svg</normaloff>:/img/stop.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../common/resources/common.qrc"/>
 </resources>
 <connections/>
</ui>
